vllm/model_executor/model_loader.py

```diff
@@ -77,7 +80,11 @@ def get_model(model_config: ModelConfig, device_config: DeviceConfig,
         # Create a model instance.
         # The weights will be initialized as empty tensors.
         with torch.device(device_config.device):
-            if hasattr(model_class, "supported_lora_modules"):
+            if model_config.load_format == "tensorizer" and _is_vllm_model(
+                    model_config):
+                model = load_with_tensorizer(model_class, model_config)
+                return model.eval()
```

Comment:
we should either support LoRAs and different linear methods here, or throw exceptions if they are specified to be used with tensorizer.

I am trying out LoRA support, btw